components/core/src/clp/ir/utils.hpp

```diff
@@ -3,12 +3,31 @@
 
 #include <string_view>
 
+#include "LogEvent.hpp"
+
 namespace clp::ir {
 /**
  * @param buf
  * @return Whether the content in the buffer starts with one of the IR stream magic numbers
  */
 auto has_ir_stream_magic_number(std::string_view buf) -> bool;
+/**
+ * @param buf
+ * @return Extension name of IR
+ */
+auto get_ir_extension_name() -> std::string;
```

Comment:
Do we need this method? Can we just create a constant?

Reply:
Sure, it works. it looks like I should create a new Constant.hpp file under ir/ directory, or do you have any suggestion?

Reply:
I don't have any preference. @kirkrodrigues shall we create a new file?